src/components/ModalContents/EndOfGameModalContent.js

```diff
@@ -31,10 +31,12 @@ const EndOfGameModalContent = (gameType, answersListPlayer, answersListComputer)
   };
 
   const showResults = () => {
-    const message = `Czystość Twoich wyników wynosi ${getPointsInPercentage(answersListPlayer)}%.
-    Zająłeś ${getPlaceFromHistoryRank(getPoints(answersListPlayer)) + 1} miejsce w rankingu.
-    Konkurencjny dealer uzyskał w tym czasie ${getPointsInPercentage(answersListComputer)}%.
-    Jak się nazywasz?`;
+    const message = `The purity of your results is ${getPointsInPercentage(answersListPlayer)}%.
```

Comment:
Jak rozumiem użycie tutaj słowa "purity" jest użyte w nawiązaniu do czystości produktu w serialu? 😄 
Bo jeśli nie to napisalbym zamiast tego `const message = "You answered ${getPointsInPercentage(answersListPlayer)}% questions correctly`.

Reply:
Zgadza się. To ma być nawiązanie do serialu. Jeśli jest to poprawne, to zostawię tak, jak jest.